plugins/mumble_plugin_win32.h

```diff
@@ -105,16 +154,25 @@ static bool inline initialize(const std::multimap<std::wstring, unsigned long lo
 	if (!dwPid)
 		return false;
 
+#ifndef USE_PLUGIN_LEGACY_PTR
+	static VOID *pModule;
+#endif
 	pModule=getModuleAddr(modname ? modname : procname);
 	if (!pModule) {
 		dwPid = 0;
 		return false;
 	}
+	pModule32 = reinterpret_cast<uintptr32_t>(pModule);
```

Comment:
Would it make sense to only set the corresponding ```pModuleXX``` variable if the process is 32/64bit? That way we wouldn't need to rely on different process names being available for 64/32 bit versions of games and it would be easy to detect whether you attached to something of unexpected bitness.